packages/analytics/dashboard-renderer/src/types/dashboard-renderer-types.ts

```diff
@@ -115,6 +115,14 @@ export const timeseriesChartSchema = {
     stacked: {
       type: 'boolean',
     },
+    threshold: {
+      type: 'object',
+      properties: Object.fromEntries(
+        exploreAggregations.map((key) => [key, { type: 'number' }]),
+      ),
+      required: exploreAggregations,
```

Comment:
Does this cause it to be required?

Reply:
I think its purpose is to enforce the type of the keys, not to make `threshold` itself required:
<img width="678" alt="Screenshot 2024-12-04 at 08 09 09" src="https://github.com/user-attachments/assets/4bc3c801-89ea-48e4-8b96-b0755e9bca48">